src/AppInstallerCLIE2ETests/AppInstallerCLIE2ETests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>

```

Comment:
Oops. Changes in this file are from an attempt to re-use the existing tests. Will revert.